water/main.py

```diff
@@ -61,15 +79,31 @@ def get_node_outlines(start_node):
 def update_colors(attrname, old, new):
     """Update pollution data used for node colors"""
     start_node = pollution_location_dropdown.value
+    try:
+        node_highlight = node_highlight_dropdown.value
+        node_type_highlight = node_type_dropdown.value
+    except NameError:
+        node_highlight = None
+        node_type_highlight = None
```

Comment:
Is the problem here that `update_colors` can be called before `node_highlight_dropdown` is defined?

If so, this seems like a bad workaround.